app/assets/javascripts/miq_application.js

```diff
@@ -320,7 +320,10 @@ function miqCheckForChanges() {
     }
   }
   // use default browser reaction for onclick
-  return true;
+  if(obj && (href = obj.getAttribute("href")))
```

Comment:
@himdel I did a bit more testing around this and noticed that when you jump to a new url, the existing template which is not applicable anymore (in the ```"otherwise"``` state) vanishes for a few seconds before the new page is loaded.
Other than this aesthetic side-effect, it works well.